utils/swift_build_support/swift_build_support/targets.py

```diff
@@ -12,60 +12,115 @@
 import platform
 
 
-class StdlibDeploymentTarget(object):
+class Platform(object):
+    """
+    Abstract representation of a platform Swift can run on.
+    """
+
+    def __init__(self, name, archs, sdk_name=None, is_darwin=False):
+        """
+        Create a platform with the given name and list of architectures.
+        """
+        self.name = name
+        self.targets = [Target(self, arch) for arch in archs]
+        self.sdk_name = name.upper() if sdk_name is None else sdk_name
```

Comment:
I wouldn't try to guess an SDK name like that. `sdk_name` shouldn't have a default.